datasets/fhad/make_data.py

```diff
@@ -82,10 +90,14 @@ class Sample:
 def main():
     # Setup Paths
     root_path = Path(ROOT)
+    sys.stdout.write(f"root_path is {root_path} which is a file: {os.path.isfile(root_path)}\n\n")
```

Comment:
It's silly but the convention for this is to be two lines and not directly reference variable names.   Something like this:

```python
print("Root Path is: {root_path}")
print("Does Root Path exist?: {root_path.exists()}")
```

`print()` and `sys.stdout.write()` do the same thing here.   Any reason you are using the stdout instead?  If you need extra features the logging module is a better choice then `sys.stdout.write`.   It is almost never used in python.  (usually a sign that someone comes from another language)

Don't be afraid of Pathlib.   It makes things way cleaner.   `(os.path.isfile(root_path)` == `root_path.exists()`